spec/i18n_spec.rb

```diff
@@ -292,7 +292,11 @@ def allowed_untranslated_key?(locale, key)
       I18n::Tasks::BaseTask::ALLOWED_UNTRANSLATED_KEYS.reject { |key| key[:used] }
     expect(unused_allowed_untranslated_keys).to(
       be_empty,
-      "unused allowed untranslated i18n keys: #{unused_allowed_untranslated_keys}",
+      """
+      ALLOWED_UNTRANSLATED_KEYS contains unused allowed untranslated i18n keys.
+      The following keys can be removed from ALLOWED_UNTRANSLATED_KEYS:
+      #{unused_allowed_untranslated_keys}
+      """,
```

Comment:
what is this, python?? we can get rid of leading spaces with:
```suggestion
      <<~EOS,
      ALLOWED_UNTRANSLATED_KEYS contains unused allowed untranslated i18n keys.
      The following keys can be removed from ALLOWED_UNTRANSLATED_KEYS:
      #{unused_allowed_untranslated_keys}
      EOS
```

Reply:
maybe we even upgrade to:
```
#{unused_allowed_untranslated_keys.pretty_inspect}
```

Reply:
To me, it's Elixir 😂

Reply:
I added pretty_inspect in cf1c5c9 as well